### ISSUE TITLE
Repo not whitelisted

### ISSUE THREAD

**Issue by kieferrm** [microsoft/vscode#99291]
Testing #97578.

Running on `https://online-ppe.core.vsengsaas.visualstudio.com/environments` (internal only) which run 

Version: 1.46.0-insider
Commit: 595d2b9b916d063aadfc2c1a8c8f6df5b45572a8
Date: 2020-06-03T05:23:16.078Z
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.13 Safari/537.36 Edg/84.0.522.5

1) Create a new codespace  for `https://github.com/kieferrm/js-debug-sample`. 
2) In `js-debug-sample/main.js` add the following line
```
// go to https://github.com/kieferrm/js-debug-sample/blob/master/main.js
```
3) Click the link
-> Trusted domain dialog comes up, but should not

This works fine in the desktop version.
It works as expected when I sign into GH.

**Comment by JacksonKearl** [comment]
This should be fixed with #99063.

**Comment by JacksonKearl** [comment]
\closedWith 5052292bdfac6ec6cdb179cfd5295aa38082ff1e
